src/keys/bip39.rs

```diff
@@ -190,14 +246,27 @@ impl AsRef<[u8; 64]> for Seed {
 }
 
 impl Seed {
-    pub fn null() -> Self {
+    pub fn new() -> Self {
```

Comment:
`Seed::new` may feel like it's generating a random one, at least `null` was explicit :/

Reply:
I agree, why changing it back? Wasn't `null()` just being introduced?